addons/mail/static/src/messaging/component/composer_text_input/composer_text_input.xml

```diff
@@ -4,85 +4,20 @@
     <t t-name="mail.messaging.component.ComposerTextInput" owl="1">
         <div class="o_ComposerTextInput">
             <t t-if="composer">
-                <textarea class="o_ComposerTextInput_textarea" t-esc="composer.textInputContent" placeholder="Write something..." t-on-keydown="_onKeydownTextarea" t-on-input="_onInputTextarea" t-ref="textarea" />
+                <div class="dropup o_ComposerTextInput_mentionDropdown" t-att-class="{ 'o-lowPosition': props.mentionsLowPosition }">
+                    <div class="o_ComposerTextInput_mentionDropdownPropositionList dropdown-menu" t-att-class="{ 'o-lowPosition': props.mentionsLowPosition, 'show': state.isMentionsOpen }">
+                        <t t-foreach="composer.mentionSuggestions" t-as="mention" t-key="mention.id">
+                            <PartnerMentionSuggestion
+                                    composerLocalId="composer.localId"
+                                    partnerLocalId="mention.localId"
+                                    t-on-o-partner-mention-suggestion-clicked="_onClickMentionItem"
+                            />
```

Comment:
There is an incorrect double indent here I think.

Reply:
Done